eNMS/forms/__init__.py

```diff
@@ -35,7 +35,10 @@ def __new__(cls, name, bases, attrs):
         form = type.__new__(cls, name, bases, attrs)
         if not hasattr(form, "custom_properties"):
             form.custom_properties = {}
-        form.custom_properties.update(app.properties["custom"].get(form_type, {}))
+        form.custom_properties = {
+            **form.custom_properties,
+            **app.properties["custom"].get(form_type, {}),
+        }
```

Comment:
During testing, I found adding the workflow custom properties was accidentally also modifying the service custom properties.  Merging the update into a new dictionary seemed to address this

Reply:
looks good, thanks !